data/102/049/907/102049907.geojson

```diff
@@ -5,17 +5,38 @@
     "edtf:cessation":"uuuu",
     "edtf:inception":"uuuu",
     "geom:area":0.096989,
-    "geom:area_square_m":776240019.266486,
-    "geom:bbox":"5.147115,49.497238,5.74301,49.7905235",
+    "geom:area_square_m":776239347.421883,
+    "geom:bbox":"5.147115,49.497238,5.74301,49.790523",
     "geom:latitude":49.665582,
     "geom:longitude":5.488689,
     "iso:country":"BE",
+    "label:deu_x_preferred_longname":[
+        "Bezirk Virton"
+    ],
+    "label:deu_x_preferred_placetype":[
+        "bezirk"
+    ],
     "label:eng_x_preferred_longname":[
-        "Virton Arrondissement"
+        "Arrondissement of Virton"
```

Comment:
This is either grammatically wrong or is practically too long for the label.

Reply:
The old value seems more appropriate. But prove me wrong!

Applies to all these records in the diff.

Reply:
The "Arrondissement of *" format is coming from Wikidata and Wikipedia.. pages for this specific example below:

- https://www.wikidata.org/wiki/Q93860
- https://en.wikipedia.org/wiki/Arrondissement_of_Virton

I'm open to changing these values, but I wasn't able to verify on any source (wiki, geonames, government websites, or even social media / travel sites) that the "name + Arrondissement" format was used for these place names.. so I went with this format (storing the existing as a variant).

I can easily swap these values around in a new commit, if needed.

Reply:
Hmm, okay.

I'm okay going with your suggestion but monitor customer feedback.